homeassistant/components/recorder/migration.py

```diff
@@ -2230,28 +2229,8 @@ def do_migrate(self, instance: Recorder, session: Session) -> None:
         else:
             self.migration_done(instance, session)
 
+    @retryable_database_job("meep", method=True)
```

Comment:
meep?

Reply:
Oops, that was not meant to be committed :facepalm: 

Do you think it's good enough have the same description for all data migratory jobs? Another idea is to make it `self.__class__.__name__` for methods

Reply:
Its probably fine to have the same for all of them since we are likely to get a full trace anyways when it fails